content/en/getting_started/_index.md

```diff
@@ -5,6 +5,15 @@ disable_sidebar: true
 aliases:
     - /overview
     - /getting_started/faq/
+further_reading:
+    - link: 'https://learn.datadoghq.com/course/view.php?id=18'
+      tag: 'Learning Center'
+      text: 'Introduction to Datadog'
+cascade:
+    algolia:
+        rank: 70
+        category: Getting Started
+        subcategory: Documentation
```

Comment:
Can we just remove the subcategory? Or put a different subcat on each GS page?

Reply:
If we remove the subcategory, the subcategory will default to the [page title](https://datadoghq.atlassian.net/wiki/spaces/WEB/pages/2898167073/Documentation+Search+2.0+WIP#Category-%26-Subcategory).